integration/benchmark/worker_stats_tracker.go

```diff
@@ -1,48 +1,84 @@
 package benchmark
 
 import (
-	"fmt"
+	"context"
 	"sync"
 	"time"
 
-	"github.com/jedib0t/go-pretty/table"
+	"github.com/VividCortex/ewma"
+	"github.com/rs/zerolog"
 )
 
-type WorkerStats struct {
-	workers     int
-	txsSent     int
-	txsExecuted int
+type workerStats struct {
+	workers         int
+	txsSent         int
+	txsExecuted     int
+	txsSentEWMA     float64
```

Comment:
ewma is not meaningful acronym for most ppl.  use a more descriptive abbreviation or type out the full name